scripts/systemd/nutshutdown.in

```diff
@@ -1,2 +1,14 @@
 #!/bin/sh
-@SBINDIR@/upsmon -K >/dev/null 2>&1 && @SBINDIR@/upsdrvctl shutdown
+
+if @SBINDIR@/upsmon -K >/dev/null 2>&1; then
+  wait_delay=`/bin/sed -ne 's#^ *POWEROFF_WAIT= *\(.*\)$#\1#p' /etc/nut/nut.conf`
+
+  @SBINDIR@/upsdrvctl shutdown
+
+  if [ -n "$wait_delay" ] ; then
+    sleep $wait_delay
+    /bin/systemctl reboot --force --force
```

Comment:
For posterity (so someone does not remove a "duplicate" `--force`, a comment would be welcome.